### PR TITLE
feat: add automotus org and data source

### PR DESCRIPTION
**Pull request by zanebclark**

### CODE REVIEW

**Comment by zanebclark** [comment]
@googlebot I signed it!

**Comment by zanebclark** [comment]
@pfrisella , a review at your earliest convenience would be greatly appreciated.

File: sources/automotus_insights.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "id": "AUTOMOTUS_INSIGHTS",
+  "name": "Automotus Insights",
+  "categories": ["ANALYTICS", "TIME_SERIES"],
+  "organization": "automotus_inc",
```
Comment (pfrisella) [inline sources/automotus_insights.json:5]: AUTOMOTUS_INC

**Review by pfrisella — CHANGES_REQUESTED** [review]

File: sources/automotus_insights.json (hunk 1 above)
Comment (zanebclark) [reply sources/automotus_insights.json]: 🤦 

Great catch. I've addressed this

**Comment by zanebclark** [comment]
@pfrisella , this one should be good to go now. Thanks for your review. 

Quick question for you: 
Is this the right place to file bugs and feature requests for community connectors? 
https://issuetracker.google.com/issues?q=componentid:345845%20status:open#

Or, is this a more appropriate location:

**Review by pfrisella — APPROVED** [review]
LGTM. Thanks for adding!

**Comment by pfrisella** [comment]
> @pfrisella , this one should be good to go now. Thanks for your review.
> 
> Quick question for you:
> Is this the right place to file bugs and feature requests for community connectors?
> https://issuetracker.google.com/issues?q=componentid:345845%20status:open#
> 
> Or, is this a more appropriate location:
> https://support.google.com/datastudio/community?hl=en

The issue tracker. This is also covered in the docs: https://developers.google.com/datastudio/connector/support

**Pull request merged**